scripts/release/bug_bash.py

```diff
@@ -0,0 +1,153 @@
+#!/usr/bin/env python3
+# Copyright (c) 2021, Facebook
+#
+# SPDX-License-Identifier: Apache-2.0
+
+"""Query the Top-Ten Bug Bashers
+
+This script will query the top-ten Bug Bashers in a specified date window.
+
+Usage: ./scripts/bug-bash.py -t <gh_token> -b 2021-07-26 -e 2021-08-07
+"""
+
+import argparse
+from datetime import datetime, timedelta
+from github import Github
+import operator
+
+
+def parse_args():
+    parser = argparse.ArgumentParser()
+    parser.add_argument('-t', '--token', dest='token',
+                        help='GitHub token', metavar='token', required=True)
```

Comment:
IMO it's bad practice to encourage leaving credentials in .bash_history. I would rather see something like this:

https://github.com/nrfconnect/sdk-nrf/blob/0fc8888738cbdc810d43d85402255f9fe5421b95/scripts/get_pulls_in_range.py#L150

I just noticed that script is missing a copyright and license, so I added one here:

https://github.com/nrfconnect/sdk-nrf/pull/5320

Once that is merged you should just be able to copy/paste into here if you want.

Reply:
Good call - I may do something similar where there is an option to specify a credentials file and then fallback to `GITHUB_TOKEN` / error if unset.